tutorials/test_RobustLoss.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
For the unit test, the right place to do it is [this](https://github.com/facebookresearch/theseus/blob/main/tests/core/test_robust_cost.py) file. There is no need to write a full example. We only need two versions of a small objective and compare that the both versions lead to the same error vectors. We can remove this notebook.

Reply:
I use notebook to test to avoid the old version will mess up with the new one. Because I have installed Theseus 0.1.4 and do not want to make any changes to it. And any of the function that call th.xx will call the original Theseus function. Do you have any good ideas? Or I can just replace my new script file with the one in the environment site-package?